src/hooks/index.ts

```diff
@@ -115,3 +115,8 @@ export const useUserFollowing = () : UserFollowing => {
 
     return { following, onFollow, onUnfollow };
 };
+
+export const useFocusDate = (): [Date, Dispatch<SetStateAction<Date>>] => {
```

Comment:
The type of `setFocusDate()` seems unnecessarily complex. Wouldn't just `(Date) => void` do?